tests/Unit/ThirdParty/Plugins/Smush.php

```diff
@@ -0,0 +1,86 @@
+<?php
+namespace WP_Rocket\Tests\Unit\ThirdParty\Plugins\Smush;
+
+use WP_Rocket\Subscriber\Third_Party\Plugins\Smush_Subscriber;
+use WP_Rocket\Tests\Unit\TestCase;
+use Brain\Monkey\Functions;
+
+class Smush extends TestCase
+{
+    /**
+     * Setup constants required by Smush plugin & include the smush.php
+     *
+     * @since 3.4.2
+     * @author Soponar Cristina
+     *
+     * @return void
+     */
+    protected function setUp()
+    {
+        parent::setUp();
+
+        if ( ! defined( 'WP_SMUSH_VERSION' ) ) {
+            define( 'WP_SMUSH_VERSION', '3.2.4' );
+        }
+        if ( ! defined( 'WP_SMUSH_PREFIX' ) ) {
+            define( 'WP_SMUSH_PREFIX', 'wp-smush-' );
+        }
+    }
+
+    /**
+     * Test should disable WP Rocket lazy load functionality when Smush lazyload is enabled
+     *
+     * @since 3.4.2
+     * @author Soponar Cristina
+     *
+     */
+    public function testShouldDisableWPRocketLazyLoad()
+    {
+        $subscriber  = new Smush_Subscriber();
+
+        Functions\expect( 'get_option' )
+            ->once() // called once
+            ->andReturn( [ 'lazy_load' => true ] );
+
+        Functions\when( '__' )
+            ->justReturn( 'Smush, ' );
```

Comment:
This should not be needed with Brain Monkey, the library takes care of this kind of WP functions automatically

Reply:
If I remove this I receive this error:  `"__" is not defined nor mocked in this test.`
 https://www.screencast.com/t/ciaLlNinxk

Reply:
Ah right, in fact it's Greg who wrote something to handle this. At the start of the method, you can use `$this->mockCommonWpFunctions();` and it will handle it for you 👍